cmd/get/kubeconfig.go

```diff
@@ -40,6 +41,11 @@ func NewKubeconfigCmd() *cobra.Command {
 		PreRun: func(cmd *cobra.Command, _ []string) {
 			cmdEvent = analytics.NewCommandEvent(cobrax.GetFullname(cmd))
 
+			// Load and validate flags from configuration FIRST.
+			if err := flags.LoadAndMergeCommandFlags("get"); err != nil {
+				logrus.Fatalf("failed to load flags from configuration: %v", err)
+			}
+
 			if err := viper.BindPFlags(cmd.Flags()); err != nil {
 				logrus.Fatalf("error while binding flags: %v", err)
 			}
```

Comment:
shouldn't this be the other way around? 
first load the flags with viper and then if they are not loaded, load them from config file

maybe it's not so much of a problem because `viper.BindPFlags` may override them, but it's better to be safe and do flags.LoadAndMergeCommandFlags later

```suggestion

			if err := viper.BindPFlags(cmd.Flags()); err != nil {
				logrus.Fatalf("error while binding flags: %v", err)
			}
			
			// Load and merge flags from configuration file.
			if err := flags.LoadAndMergeCommandFlags("get"); err != nil {
				logrus.Fatalf("failed to load flags from configuration: %v", err)
			}
```